pages/api/unsubscribe.tsx

```diff
@@ -18,7 +18,7 @@ import { DocumentReference } from "firebase-admin/firestore";
 import { verifyAdminToken, verifyAuthHeader } from "@/lib/api-helpers/auth";
 
 async function generateUnsubKey(docRef: DocumentReference): Promise<string> {
-  const unsubKey = crypto.randomBytes(32).toString("hex");
+  const unsubKey = crypto.randomBytes(6).toString("hex");
```

Comment:
What do we gain from the length? Trying to avoid the crazy long url. I imagine there are diminishing returns with how long this value is.